app/components/article-comments.js

```diff
@@ -122,42 +95,4 @@ export default Component.extend({
   scrollTop() {
     scrollToTop(this.element);
   },
-
-  fetchComments(page) {
```

Comment:
@xkxd does it mean we are not supporting comments from app once this change is released?